matcher/service/openreview_interface.py

```diff
@@ -96,11 +98,35 @@ def maximums(self):
 
         return self._maximums
 
+    @property
+    def custom_demand_edges(self):
```

Comment:
if these methods are not going to be used outside of the interface we don't need to set them as a property, they should 'private' methods.

Reply:
You are right, these methods can be private. I have removed the property decorator and made these methods as private.

But just a side note: Setting these methods as a property is just a convenience feature that allows us to call them as an object property instead of a method and I do not think that it has anything to do with the methods being private. 
Basically, they can be private methods and can still utilize the property decorator.

Reply:
Please see my recent comment.

We already have minimuns, maximums and demands as properties. Edges should be retrieved only once to compute these properties.

Reply:
Fixed.